docs/Developer Reference/create-a-chainlinked-project.md

```diff
@@ -89,11 +89,43 @@ cd MyChainlinkProject/
 truffle unbox smartcontractkit/truffle-starter-kit
 ```
 
-For instructions about how to use the starter kit, see the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).
+For instructions on how to use the starter kit, refer to the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).
 
 For more details on how to use Chainlink with Truffle, see our blog post [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).
 ___
 
+## DappTools Starter Kit
+
+To learn more about DappTools, refer to the [DappTools Documentation](https://dapp.tools/). You can also refer to the [Installation instructions](https://github.com/dapphub/dapptools#installation).
+
+Once installed, clone the starter kit and install the project's dependencies:
+
+```shell
+git clone https://github.com/smartcontractkit/dapptools-starter-kit
+cd dapptools-starter-kit
+make # This installs the project's dependencies.
+```
+
+For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).
+
+For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
+___
+
+## Foundry Starter Kit
+
+To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/). You you also refer to the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html).
+
+Once installed, clone the starter kit and install the project's dependencies:
+
+```shell
+git clone https://github.com/smartcontractkit/foundry-starter-kit
+cd foundry-starter-kit
+make # This installs the project's dependencies.
+```
+
+For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).
```

Comment:
```suggestion
To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/).

1. Install Foundry using the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html) on GitHub.io.

1. After you install Foundry, clone the starter kit and install the projects dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/foundry-starter-kit
      cd foundry-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).
```